flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/client/HiveShimV1.java

```diff
@@ -79,4 +85,14 @@ public Function getFunction(IMetaStoreClient client, String dbName, String funct
 			throw e;
 		}
 	}
+
+	@Override
+	public boolean moveToTrash(FileSystem fs, Path path, Configuration conf, boolean purge) throws IOException {
```

Comment:
how is the "purge" flag taking effects?

Reply:
Hive-1.2.1 doesn't support auto purge for insert overwrite, see [HIVE-15880](https://issues.apache.org/jira/browse/HIVE-15880). So it doesn't have any effect for HiveShimV1.

Reply:
I see, can we add a comment to the interface's param explaining the flag only works for hive 2 (or 2.1.x, or any version it works)?